From: =?utf-8?q?Jeremy_B=C3=ADcha?= <jeremy.bicha@canonical.com>
Date: Thu, 21 Mar 2024 17:22:34 -0400
Subject: online accounts: sort

Sort Nextcloud below Google & Microsoft since that's a
more useful order for our users
---
 panels/online-accounts/cc-online-accounts-panel.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/panels/online-accounts/cc-online-accounts-panel.c b/panels/online-accounts/cc-online-accounts-panel.c
index daf212e..85a0de4 100644
--- a/panels/online-accounts/cc-online-accounts-panel.c
+++ b/panels/online-accounts/cc-online-accounts-panel.c
@@ -290,11 +290,11 @@ static int
 goa_provider_priority (const char *provider_type)
 {
   static const char *goa_priority[] = {
-    "owncloud",     /* Nextcloud */
     "google",       /* Google */
     "windows_live", /* Microsoft Personal */
     "ms_graph",     /* Microsoft 365 */
     "exchange",     /* Microsoft Exchange */
+    "owncloud",     /* Nextcloud */
     "fedora",       /* Fedora */
     "imap_smtp",    /* Email (IMAP and SMTP) */
     "webdav",       /* Calendars, Contacts, Files (WebDAV) */
